modules/galera/detectors-galera.tf

```diff
@@ -0,0 +1,119 @@
+    detect_label          = "CRIT"
+    disabled              = coalesce(var.wsrep_local_state_disabled, var.detectors_disabled)
+    notifications         = coalescelist(lookup(var.wsrep_local_state_notifications, "critical", []), var.notifications.critical)
+    parameterized_subject = local.rule_subject
+    parameterized_body    = local.rule_body
+  }
+}
+
+resource "signalfx_detector" "wsrep_flow_control_paused" {
+  name = format("%s %s", local.detector_name_prefix, "MySQL Galera flow_control_paused")
+
+  program_text = <<-EOF
+    signal = data('mysql_wsrep_flow_control_paused', filter=${module.filter-tags.filter_custom})${var.wsrep_flow_control_paused_aggregation_function}${var.wsrep_flow_control_paused_transformation_function}.publish('signal')
+    detect(when(signal > ${var.wsrep_flow_control_paused_threshold_critical})).publish('CRIT')
+    detect(when(signal > ${var.wsrep_flow_control_paused_threshold_major}) and when(signal < ${var.wsrep_flow_control_paused_threshold_critical})).publish('MAJOR')
+EOF
+
+  rule {
+    description           = "is > ${var.wsrep_flow_control_paused_threshold_critical}%"
+    severity              = "Critical"
+    detect_label          = "CRIT"
+    disabled              = coalesce(var.wsrep_flow_control_paused_disabled_critical, var.wsrep_flow_control_paused_disabled, var.detectors_disabled)
+    notifications         = coalescelist(lookup(var.wsrep_flow_control_paused_notifications, "critical", []), var.notifications.critical)
+    parameterized_subject = local.rule_subject
+    parameterized_body    = local.rule_body
+  }
+
+  rule {
+    description           = "is > ${var.wsrep_flow_control_paused_threshold_major}%"
+    severity              = "Major"
+    detect_label          = "MAJOR"
+    disabled              = coalesce(var.wsrep_flow_control_paused_disabled_major, var.wsrep_flow_control_paused_disabled, var.detectors_disabled)
+    notifications         = coalescelist(lookup(var.wsrep_flow_control_paused_notifications, "major", []), var.notifications.major)
+    parameterized_subject = local.rule_subject
+    parameterized_body    = local.rule_body
+  }
+}
+
+resource "signalfx_detector" "wsrep_local_recv_queue_avg" {
+  name = format("%s %s", local.detector_name_prefix, "MySQL Galera wsrep_local_recv_queue_avg")
+
+  program_text = <<-EOF
+    signal = data('mysql_wsrep_local_recv_queue_avg', filter=${module.filter-tags.filter_custom})${var.wsrep_local_recv_queue_avg_aggregation_function}${var.wsrep_local_recv_queue_avg_transformation_function}.publish('signal')
+    detect(when(signal > ${var.wsrep_local_recv_queue_avg_threshold_critical})).publish('CRIT')
+    detect(when(signal > ${var.wsrep_local_recv_queue_avg_threshold_major}) and when(signal < ${var.wsrep_local_recv_queue_avg_threshold_critical})).publish('MAJOR')
+EOF
+
+  rule {
+    description           = "is too high > ${var.wsrep_local_recv_queue_avg_threshold_critical}"
+    severity              = "Critical"
```

Comment:
every detectors are critical here may be this one could be decreased with Major / Minor rules

Reply:
Agree, changed.